src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
This should follow the [duck pattern](https://github.com/erikras/ducks-modular-redux) e.g `const CHECK_STATUS = bookstore/books/CHECK_STATUS`